web/docs/language/features.md

```diff
@@ -935,6 +939,82 @@ import AuthError from '@wasp/core/AuthError.js'
   }
 ```
 
+### Google
+
+`google` authentication makes it possible to use Google's OAuth 2.0 service to sign Google users into your app. To enable it, add `google: {}` to your `auth.methods` dictionary to use it with default settings. If you require custom configuration setup or a sign-in function, you can override the defaults.
+
+#### Default settings
+- Configuration:
+  - By default, we expect you to set two environment variables in order to use Google authentication:
+    - `GOOGLE_CLIENT_ID`
+    - `GOOGLE_CLIENT_SECRET`
+  - These can be obtained in your Google Cloud Console project dashboard. See [here](/docs/integrations/google#google-auth) for more.
+
+- Sign in:
+  - By default, when a user signs in with Google we will attempt to look up the user by email.
+    - If they exist, we will do nothing and return the JWT.
```

Comment:
Aha, so this is sign in! Is there a difference between sign in and "sign up"? Not really right? It doesn't matter if it is the first time we are signing in with Google or 10th time, if user already exists from before?

Reply:
Yeah, there really is no difference from the user of the web app's perspective. They just "sign in." Behind the scenes, we decide if they need a new User or not is all.